facades/PC/build.gradle.kts

```diff
@@ -111,6 +101,22 @@ configurations {
         description = "for fetching modules for running a server"
         isTransitive = false
     }
+    register("subsystem") {
+        description = "for fetching modules for running a server"
+    }
+    implementation {
+        extendsFrom(configurations["subsystem"])
+    }
+}
+
+dependencies {
+    implementation(project(":engine"))
+    "subsystem"(project(":subsystems:DiscordRPC"))
```

Comment:
Yeah, it is configuration.
In groovy i can omit `"`. 
But it is kotlin. and this configuration from this file, not from plugins.

Reply:
It would be nice to complete #4157 before introducing new complications to the facade build.

I think I would have stuck with the built-in `implementation` Configuration for the subsystems here instead of making a custom Configuration for them.

I can start to imagine why it might be nice to separate them. For example, if we had a lot of Subsystems, and a bunch of other direct dependencies, and we had some troubleshooting use case where we needed to see the Subsystem dependencies independently and there was a nice diagnostic report for that —

but I don't expect that to be the case, and I'm concerned it's not worth the added complexity to the build file and the loss of edit-time type-checking on the custom Configuration name. Are there other benefits to it I have overlooked?

Reply:
I don't think that picocli will produce many conflicts.

New configuration introduced for classpath at run tasks (you made building classpath from engine, it's dependencies and facade )

Reply:
> you made building classpath from engine, it's dependencies and facade

I didn't! But yes my name is probably on all the lines because of the change to `.kts`.

> I don't think that picocli will produce many conflicts.

Hmm, I see I was unclear when I referred to picocli's issue. The bug that is blocking the picocli PR is that picocli isn't included in the runtime classpath of the `game` task (and others). That is why I have been asking all [these](https://github.com/MovingBlocks/Terasology/pull/4157#issuecomment-704658753) [questions](https://github.com/MovingBlocks/Terasology/pull/4157#issuecomment-716830609) about why the heck those tasks are written the way they are.

But this is not really in the scope of picocli, and I believe you have run in to the same problem here, let us separate it to another issue: #4238 

My objection is that these tasks already have a bunch of configuration in them that nobody seems to be able to explain, and this change makes the facade build configuration more complicated, not less.

So it's not that this conflicts with picocli, exactly. It conflicts with trying to fix the bug-that-blocks-picocli.